app/views/devise/shared/_password_strength.html.erb

```diff
@@ -1,4 +1,4 @@
-<div id='pw-strength-cntnr' class='hide' aria-live='polite' aria-atomic='true'>
+<div id='pw-strength-cntnr' class='display-none' aria-live='polite' aria-atomic='true'>
```

Comment:
As described in [a JavaScript comment](https://github.com/18F/identity-idp/blob/d54a6beb42a3e0fab1949d426f8f8956db98a2b4/app/javascript/packs/pw-strength.js#L131-L134), the purpose of the class is to hide the strength meter from JavaScript-disabled browsers.

Suggesting to keep this as `display: none`.